common/artifacts/901_wc_artifacts_weapon_twohanded.txt

```diff
@@ -360,6 +360,72 @@ bloodhoof_runespear = {
 	picture = "GFX_bloodhoof_runespear"
 	slot = weapon
 }
+twin_blades_of_azzinoth = {
+	quality = 5
+	flags = { fel_artifact combat_weapon sword }
+
+	active = {
+		is_adult = yes
+
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_physical_class_trigger = yes
+			is_combined_class_trigger = yes
+		}
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 1
+	intrigue = 1
+	combat_rating = 20
+
+	monthly_character_prestige = 1
+	
+	kaldorei_religion_opinion = -10
+	fel_group_opinion = 5
```

Comment:
So, for example, paladins can wield it and gain bonus opinion with the Legion?

Reply:
> So, for example, paladins can wield it and gain bonus opinion with the Legion?

Restrict them to warrior/rogue only perhaps?

Reply:
or decide what class illidan will get, especially with TFT startdate. Demonhunter as class/subclass might be necessary

Reply:
> or decide what class illidan will get, especially with TFT startdate. Demonhunter as class/subclass might be necessary

I'd still let warrior/rogue use it due to the fact those two could use them in tbc if you somehow manage to kill Illidan and steal his stuff.

Reply:
> Restrict them to warrior/rogue only perhaps?

So, scarlet light warrior will get bonus opinion with the Legion?

Reply:
> > Restrict them to warrior/rogue only perhaps?
> 
> So, scarlet light warrior will get bonus opinion with the Legion?

That wasn't my point, but to which classes can use them.

Reply:
Gosh, just restict the blades to `religion_group = fel_group`.

Reply:
Done.